python/jpyinterpreter/setup.py

```diff
@@ -15,9 +15,9 @@ class FetchDependencies(build_py):
     """
     def create_stubs(self, project_root, command):
         subprocess.run([str((project_root / command).absolute()), 'dependency:copy-dependencies'],
-                       cwd=project_root, check=True)
+                       '-Dpython', '-Dquickly', '-Dexec.skip', cwd=project_root, check=True)
```

Comment:
This could go into the special Maven profile, no? That way, the config would be shared without having to repeat it.